src/main/java/com/algorithmtron/controller/FindFirstAndLastIndexOFElementSortedArrayController.java

```diff
@@ -0,0 +1,32 @@
+package com.algorithmtron.controller;
+
+import com.algorithmtron.service.FindFirstAndLastIndexOfElementSortedArrayService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import static com.algorithmtron.utils.Utils.convertStringToIntArray;
+
+@Controller
+public class FindFirstAndLastIndexOFElementSortedArrayController {
```

Comment:
Could this name be shortened a bit? Perhaps "positionFinder" or smething like that?

Reply:
Well in that case it wouldnt be so explicit about which positions the controller is going to find. But if you consider it an annoyance in the wording as it is right now, i'll change it. I'll await your response.

Reply:
Well, the naming of the methods should represent what it does. Think about the length of the url when you will call this endpoint :D And if you will have more added to it, it may be the case you won't be able to see it in the browser anymore.

Reply:
You can see more info here : https://blog.dreamfactory.com/best-practices-for-naming-rest-api-endpoints/

Reply:
Fair point! positionFinder sounds like a good name in this case. would you agree?

Reply:
Yes, sound wonderfull :D